xds/src/main/java/io/grpc/xds/ClusterImplLoadBalancer.java

```diff
@@ -317,10 +317,13 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
                   "Cluster max concurrent requests limit exceeded"));
             }
           }
+          ClientStreamTracer.Factory tracerFactory = result.getStreamTracerFactory();
           final ClusterLocalityStats stats =
               result.getSubchannel().getAttributes().get(ATTR_CLUSTER_LOCALITY_STATS);
-          ClientStreamTracer.Factory tracerFactory = new CountingStreamTracerFactory(
+          if (stats != null) {
```

Comment:
Any idea why this could be null?

Reply:
I think it is because in my test config, i didn't put lrs_server for simplicity, but im not sure if it is allowed in real world.
https://github.com/grpc/grpc-java/blob/881747a63df79a7baae252de86447c0997c76f69/xds/src/main/java/io/grpc/xds/ClientXdsClient.java#L1579